webdriverio/appium-app/examples/simple-example/test/configs/wdio.android.local.emu.conf.ts

```diff
@@ -0,0 +1,15 @@
+import {join} from 'path';
+import config from './wdio.shared.local.appium.conf';
+
+config.capabilities = [
+    {
+        platformName: 'Android',
```

Comment:
can we change this to W3C caps?

Reply:
i thought platformName was W3C? Sorry, I'm new to W3C stuff.